### ISSUE TITLE
The name, role, state, and value of all user interface components can be programmatically determined (especially when the state changes).

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#119]
The mobile site navigation toggle doesn't have a meaningful name to convey to screen-reader users. We recommend adding an aria-label of "Purdue Menu" to communicate the button's function to users.
